data/strings/strings.txt

```diff
@@ -21967,16 +21967,17 @@
     zh-Hant = 繞行設定
 
   [avoid_tolls]
+    comment = Should be maximum 25 characters
```

Comment:
comment = Recommended length for CarPlay and Android Auto is around 25-27 characters.

@AndrewShkrob can you please confirm the AA limit?

Reply:
Approximately 26 characters, but it depends on word length and line breaks.